### DISCUSSION TITLE
Help on GPR with multi-dimensional input (single dimension output)

### DISCUSSION THREAD

**Discussion by rumi1359** [cornellius-gp/gpytorch#1774] [Q&A]
Hello dev team. I'm having trouble constructing a GP regression model where train_x is multi-dimensional. Basically I have a bunch of training samples: "x" where each x has N>1 features (i.e is a 1xN tensor). The target "y" is still one dimensional for each x. The "GPyTorch Regression Tutorial" assumes both x and y to be 1 dimensional. How can I modify this for my use case? Thank you.

**Comment by gpleiss** [comment]
Your `x` tensor should be shaped `num_data x num_dimensions`, and your `y` tensor should be shaped `num_data`.

**Reply by rumi1359** [reply]
And then the same structures? : 
likelihood = gpytorch.likelihoods.GaussianLikelihood()
model = ExactGPModel(train_X, train_Y, likelihood)

**Reply by gpleiss** [reply]
Yes, same structures
